src/popup.tsx

```diff
@@ -87,7 +87,9 @@ function Popup() {
             <div className="shortlink-container">
               {allShortlinks.map((shortlink) => (
                 <code key={shortlink.name} className="shortlink">
-                  {shortlink.name}
+                  <a href={`https://www.${shortlink.name}`} target="_blank" className="shortlink-link">
```

Comment:
@JoHeyJo This actually does not work. Each shortlink name is a key. The value is a list of URLs which have to be opened when this is clicked.

Reply:
so every URL in `shorlink.urls` should be opened as a new tab? I'm also noticing the URLs stored in `urls` don't match `shortlink.name`

Reply:
This commit shows what has to happen: 
https://github.com/r3bl-org/shortlink/commit/947345e0f9106989303265bc8d71b9e1f16c9046
👍🏽